scripts/95-get_metadata.py

```diff
@@ -123,9 +123,17 @@ def get_kernel_metadata(platform: str) -> dict:
     architecture = argv[4]
     platform = argv[5]
     device = argv[6]
+
     print(f"debos_ref={debos_ref}")
     data = dict()
-    data["core"] = get_neon_core_meta(core_ref)
+
+    if image_name.startswith("debian-neon-image"):
+        edition = "Standalone"
+        data["core"] = get_neon_core_meta(core_ref, "NeonCore", "neon_core/version.py")
+    elif image_name.startswith("debian-node-image"):
+        edition = "Node"
+        data["node"] = get_neon_node_meta(core_ref, "neon-nodes", "neon_nodes/version.py")
```

Comment:
Would this be easier to parse as "core" with the "edition" meta identifying what the "core" package is?